projects/npm-tools/packages/npm-scripts/src/scripts/build.js

```diff
@@ -130,6 +144,13 @@ module.exports = async function (...args) {
 		cleanSoy();
 	}
 
+	if (!BUILD_CONFIG.disableSass) {
+		buildSass(path.join(CWD, BUILD_CONFIG.input), {
```

Comment:
Hey @bryceosterhaus, we can wrap `buildSass` in a call to `instrument` to get build-time performance metrics for it.

Basically, you use `let` when you `require` it ([like this](https://github.com/liferay/liferay-frontend-projects/blob/7a869cded3aed96f3bd72b22b464b9f18694aca9/projects/npm-tools/packages/npm-scripts/src/scripts/build.js#L12)), and then add `buildSass` to [the list of instrumented functions](https://github.com/liferay/liferay-frontend-projects/blob/7a869cded3aed96f3bd72b22b464b9f18694aca9/projects/npm-tools/packages/npm-scripts/src/scripts/build.js#L26-L48).

Demo and sample results of this in [the PR that added instrumentation](https://github.com/liferay/liferay-frontend-projects/pull/299) (and [example analysis here](https://docs.google.com/spreadsheets/d/1gV6ZDO2lkkQSd4yAf5fXM5xDwSNc3SOtO0ywORfwbh4/edit)).